docs/tutorial/category_theory/calculating_colimits_in_Top.lean

```diff
@@ -61,28 +72,32 @@ section Gluing
 
 -- Here's two copies of the real line glued together at a point.
 def f : pt ⟶ R := { val := λ _, (0 : ℝ), property := continuous_const }
+/-- Two copies of the real line glued together at 0. -/
```

Comment:
Maybe include a newline before the docstring?

Reply:
Fixed.